internal/server/data/data.go

```diff
@@ -265,20 +264,6 @@ func initialize(db *DB) error {
 	return tx.Commit()
 }
 
-func getDefaultSortFromType(t interface{}) string {
```

Comment:
did we lose default sorts?

Reply:
Every list function has a sort, but there's no central function that uses reflection to try and guess at which field to use.